docs/contributing.md

```diff
@@ -0,0 +1,10 @@
+### How can I help contribute to howdoi?
+
+If you would like to add or extend existing functionality, fix a bug or anything inbetween please follow the standard procedure for most open source projects. i.e:
+
+1. First check the issues to see if anyone is working on your issue before starting work yourself. Someone else might be working on it already or a maintainer might've closed the issue as they don't feel it would be beneficial to the project.
+2. Submit the issue on the repo and begin your work on your fork of the repo.
+3. Submit your PR once completed and wait for feedback.
+
+!!! Note
```

Comment:
For this, I would suggest adding a colon and bolding `**Note:**`. What do you think?

Reply:
I think this is a custom syntax extension to Markdown. https://squidfunk.github.io/mkdocs-material/reference/admonitions/